jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-deploy.md

```diff
@@ -0,0 +1,22 @@
+# Initialize a Worker
+
+## Overview
+
+This command is used to Deploy a worker.
```

Comment:
lower case, "deploy". Maybe say "This command is used to update the worker definition (code, description , filter, secret ...) on your Artifactory instance